### DISCUSSION TITLE
Unit not starting

### DISCUSSION THREAD

**Discussion by liselotte111** [CrashOverride85/zc95#85] [General]
After assembling everything, the unit only wants to boot with the dc adapter plugged in (the front leds were blinking red). After turning it off and on a couple of times, now it sometimes starts on battery power.  The battery is fully charged 

Maybe someone can point me in the right direction?

Edit:  turns out one of the 0.5R soldered by JLCPCB (I forgot to remove it from the output board assembly list) was cracked. Another reason not to let them solder any through hole components.

**Comment by CrashOverride85** [comment]
Interesting, I'm surprised the 0.5R resistor was affected by being on battery or external power. Is it working and starting up ok now? Or if not, is there anything showing on the screen?

**Reply by liselotte111** [reply]
Unfortunately not, it passes the self-test about 1/10 times. Otherwise it reports an error on channel 2. Ive checked the serial output, and that channel has a voltage of 0.0740-0.0749. No clear difference between running on battery or mains. It does appears to pass the test once after I've (re)connected the battery. My knowledge of electrpnics isn't enough to troubleshoot this. Im thinking of replacing that mosfet?

**Reply by CrashOverride85** [reply]
It only being one channel does rule out quite a lot of stuff which is good. Could you post the full section of the serial output where it's doing the self test? including the other channels that are working? 0.074v suggests it's _almost_ working

**Reply by liselotte111** [reply]
I replaced the fet of channel 2, no more errors. Thanks for the quick reply!
One more thing, sometimes the display turns on scrambled. It's a generic St7735 display, so that's probably the culprit?

**Reply by CrashOverride85** [reply]
Nice, glad it's working 🙂️

I'm not sure - the only way I was aware of to get a scrambled screen was to power the main pico over USB, then power cycle the box using the switch. As the pico never resets, the screen doesn't get reinitialised so is corrupted.
